src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,127 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int DEFAULT_MULTIPLIER = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
```

Comment:
this is a useless field - it's just a product of multiplication of capacity and load factor, you can get it at any time